src/Views/BucketPanel/EntryDetail.test.tsx

```diff
@@ -181,6 +197,12 @@ describe("EntryDetail", () => {
           lineNumbers: true,
         }),
       );
+
+      const cmValue = wrapper.find("Controlled").prop("value");
+      expect(cmValue).toContain("$limit");
+
+      const exampleText = wrapper.find(".jsonExample").at(0).text();
+      expect(exampleText).toContain("$limit");
```

Comment:
I've just noticed that we don't check  if Bucket.query was called. Please  add a check with the when condition.